Makefile

```diff
@@ -254,8 +254,8 @@ generate-bindata: $(KUSTOMIZE) $(GOBINDATA) clean-bindata $(CLOUDINIT_GENERATED)
 	# Package manifest YAML into a single file.
 	mkdir -p $(GOBINDATA_CLUSTERCTL_DIR)/manifest/
 	$(KUSTOMIZE) build $(GOBINDATA_CLUSTERCTL_DIR)/crd > $(GOBINDATA_CLUSTERCTL_DIR)/manifest/clusterctl-api.yaml
-	# Fetch the cert-manager manifest
-	curl -sL https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml > "$(GOBINDATA_CLUSTERCTL_DIR)/manifest/${CERTMANAGER_COMPONENTS_GENERATED_FILE}"
+	# Copy the "fixed" cert-manager v0.11.0 manifest
```

Comment:
Is it possible to keep the manifest under `cmd/clusterctl/config` eventually in a subfolder, and thus avoid the copy step?

Reply:
Sure. How about in a dir like `cmd/clusterctl/config/assets` and I can modify the gobindata cmd to include that directory. WDYT?